### ISSUE TITLE
Lightbox - Gallery traversing seems to not work anymore

### ISSUE THREAD

**Issue by matthisamoto** [Formstone/Formstone#295]
See JS error on [Demo page](https://formstone.it/components/lightbox/demo/) under Gallery, when you click the image, then the next/prev arrows to traverse through the gallery items from the lightbox.

In Chrome:
```
site.js?v=1.4.18-1:7960 

Uncaught TypeError: Cannot read property 'match' of undefined
    at checkVideo (site.js?v=1.4.18-1:7960)
    at k.fn.init.cleanGallery (site.js?v=1.4.18-1:7748)
    at resolve (site.js?v=1.4.18-1:12926)
    at HTMLDivElement.onTranistionEnd (site.js?v=1.4.18-1:12900)
    at HTMLDivElement.dispatch (site.js?v=1.4.18-1:2)
    at HTMLDivElement.v.handle (site.js?v=1.4.18-1:2)
```

In Firefox:
```
site.js:7960:11

Uncaught TypeError: source is undefined
    checkVideo https://formstone.it/demo/js/site.js?v=1.4.18-1:7960
    cleanGallery https://formstone.it/demo/js/site.js?v=1.4.18-1:7748
    resolve https://formstone.it/demo/js/site.js?v=1.4.18-1:12926
    onTranistionEnd https://formstone.it/demo/js/site.js?v=1.4.18-1:12900
    dispatch https://formstone.it/demo/js/site.js?v=1.4.18-1:2
    handle https://formstone.it/demo/js/site.js?v=1.4.18-1:2
```

**Comment by matthisamoto** [comment]
Seems like there is no variable defined for `source` in [this else block](https://github.com/Formstone/Formstone/blob/2e9170852412a50a5a2edccfe5c2d7d2bd5852f3/src/js/lightbox.js#L1260-L1275) 

It appears this happened when the defined `source` variable was moved in `src/js/lightbox.js` via [this commit](https://github.com/Formstone/Formstone/commit/2e9170852412a50a5a2edccfe5c2d7d2bd5852f3#diff-59c604e3b17aca1926e8a5175414a34f7f910553fb2496971bf18a0cf8682187)

**Comment by matthisamoto** [comment]
I am able to "fix" this by copying the `source` declaration as-is into the else block, or by just pulling it back out to where it existed previously outside the if/else.

**Comment by benplum** [comment]
Fixed in 1.4.19.
